scripts/ait_bootstrap.sh

```diff
@@ -39,10 +47,38 @@ curl -SL https://github.com/stedolan/jq/relepermanently/deleteases/download/jq-1
 chmod +x /usr/bin/jq
 }
 
+function bootstrap_data(){
+# Adding functionality to mount an EFS
+echo "Bootstrapping EFS and Mounting it"
+sudo yum -y update  
+sudo yum -y install nfs-utils
+
+# Installing EFS Amazon Utilss
+echo "Downloading and installing Amazon EFS Utils"
+sudo yum -y install git rpm-build make
+sudo git clone https://github.com/aws/efs-utils
+cd efs-utils
+sudo make rpm
+sudo yum -y install build/amazon-efs-utils*rpm
+# cd back to root dir
+cd /
+
+# Install botocore libraries
+pip3 install botocore
+# Define mount location and efs id
+DIR_TGT=/mnt/efs/
+EC2_REGION=${AWS::Region}
+EFS_FILE_SYSTEM_ID=${FileSystem}
+echo "Mounting directory to EFS: $EFS_FILE_SYSTEM_ID"
+mkdir -p $DIR_TGT
+echo "Mounting $EFS_FILE_SYSTEM_ID to $DIR_TGT"
+sudo mount -t efs  $EFS_FILE_SYSTEM_ID $DIR_TGT
+}
+
 function install_ait_and_dependents(){
 # Set variables for system install
-PROJECT_HOME=/home/ec2-user
-SETUP_DIR=/home/ec2-user/setup
+PROJECT_HOME=/mnt/efs/ait
+SETUP_DIR=/mnt/efs/ait/setup
```

Comment:
So I think that `PROJECT_HOME` should probably remain as `/home/ec2-user`; `SETUP_DIR` can probably point at EFS

But we'll still want to bootstrap everything on the _instance_ (e.g. `/home/ec2-user`)

The only tricky one here is the `AIT-Core` stuff bc it reads/writes to it "live". Just means that we will have to change later references from `$PROJECT_HOME/AIT-CORE/` to wherever AIT-CORE lives on efs

Reply:
Ok -- so when we spin up the server where do we tell it that it should now write to the EFS system?